myFitnessTrainer/src/screens/WorkoutScreen.js

```diff
@@ -1,108 +1,198 @@
-import { useState } from 'react';
+import { useState, useEffect, useRef } from "react";
 import { StyleSheet, View, Dimensions, Text } from "react-native";
-import Carousel from 'react-native-reanimated-carousel';
-import Dots from 'react-native-dots-pagination';
+import Carousel from "react-native-reanimated-carousel";
+import Dots from "react-native-dots-pagination";
 import EndWorkoutButton from "../components/EndWorkoutButton";
-import { GestureHandlerRootView } from 'react-native-gesture-handler';
-import * as Progress from 'react-native-progress';
-
-const width = Dimensions.get('window').width;
-
-// FOR TESTING, WILL USE THE DAILY EXERCISES FROM DB
-const testExercise = {
-    default_exercise_stat: null,
-    description: "this is the description",
-    equipment: "test equipment",
-    fitness_goal: "strength",
-    fitness_level: "beginner",
-    muscle_group: "lower body",
-    name: "test exercise name",
-    video: {
-        female: "www.google.com",
-        male: "www.google.com"
-    }
-};
-// FOR TESTING, USE THE DAILY EXERCISES
-const testExercisesDay0 = [testExercise,testExercise,testExercise,testExercise];
+import { GestureHandlerRootView } from "react-native-gesture-handler";
+import * as Progress from "react-native-progress";
+import {
+    getAllDocuments,
+    postDocument,
+    getDocumentId,
+} from "../../databaseFunctions";
+import SubmitExerciseStatsButton from "../components/SubmitExerciseStatsButton";
+import { auth } from "../../firebaseConfig";
+import SkipExerciseButton from "../components/skipExerciseButton";
```

Comment:
```suggestion
import SkipExerciseButton from "../components/SkipExerciseButton";
```
just a small thing to fix!